contracts/src/bridge/SequencerInbox.sol

```diff
@@ -323,9 +323,15 @@ contract SequencerInbox is DelegateCallAware, GasRefundEnabled, ISequencerInbox
      * @param ksHash hash of the keyset
      */
     function invalidateKeysetHash(bytes32 ksHash) external onlyRollupOwner {
-        if (!isValidKeysetHash[ksHash]) revert InvalidDASKeyset(ksHash);
+        if (!isValidKeysetHash[ksHash]) revert NoSuchKeyset(ksHash);
         isValidKeysetHash[ksHash] = false;
         emit InvalidateKeyset(ksHash);
         emit OwnerFunctionCalled(3);
     }
+
+    function getKeysetCreationBlock(bytes32 ksHash) external view returns (uint256) {
+        uint256 bnum = keysetHashCreationBlock[ksHash];
+        if (bnum == 0) revert NoSuchKeyset(ksHash);
+        return bnum;
```

Comment:
`keysetHashCreationBlock` is already public, so we should either make it private or remove this method